sdk/cosmos/azure-cosmos/samples/index_management_async.py

```diff
@@ -628,6 +628,54 @@ async def perform_multi_orderby_query(db):
         print("Entity doesn't exist")
 
 
+async def use_geospatial_indexing_policy(db):
+    try:
+        await delete_container_if_exists(db, CONTAINER_ID)
+
+        # Create a container with vector embedding policy and vector indexes
+        indexing_policy = {
+            'includedPaths': [
+                {'path': '/"Location"/?',
+                    'indexes': [
+                        {
+                            'kind': 'Spatial',
+                            'dataType': 'Point'
+                        }]
+                 },
+                {
+                    'path': '/'
+                }
+            ]
+        }
+
+        created_container = await db.create_container(
+            id=CONTAINER_ID,
+            partition_key=PARTITION_KEY,
+            indexing_policy=indexing_policy
+        )
+        properties = await created_container.read()
+        print(created_container)
+
+        print("\n" + "-" * 25 + "\n9. Container created with geospatial indexes")
+        print_dictionary_items(properties["indexingPolicy"])
+
+        # Create some items
+        doc9 = await created_container.create_item(body={"id": "loc1", 'Location': {'type': 'Point', 'coordinates': [20.0, 20.0]}})
```

Comment:
yep agreed, will add more tests for the other queries that can be done - for the sample however I will leave only one way to do it since the point is not to be all-exhaustive but to rather show a snippet how it may be done. If you look at most of our samples, that is the general take for them - otherwise the sample might as well be the documentation we have here:
https://learn.microsoft.com/azure/cosmos-db/nosql/how-to-manage-indexing-policy?tabs=dotnetv3%2Cpythonv4#use-the-python-sdk
https://learn.microsoft.com/azure/cosmos-db/nosql/how-to-geospatial-index-query
https://learn.microsoft.com/azure/cosmos-db/nosql/query/geospatial?tabs=javascript

Reply:
Thank you Simon, that sounds good to me.